docs/en/operation/Admin-CLI.md

```diff
@@ -69,26 +54,39 @@ $ ./bin/alluxio fsadmin doctor configuration
 
 ### report
 
-// TODO(lu) detailed description about each of the subcommands.
-// such as the report capacity usage with filter tags.
-
 The `report` command provides Alluxio running cluster information. 
 
+If no argument is passed in, `report` will report the leader master, worker number, and capacity information.
+
 ```bash
-# Report cluster summary
 $ ./bin/alluxio fsadmin report
-#
-# Report worker capacity information
+```
+
+`report capacity` will report Alluxio cluster capacity informatin of different ranges: 
+* `-live` Live workers
+* `-lost` Lost workers
+* `-workers <worker_names>` Specified workers, host names or ip addresses separated by `,`. 
+
+```bash
+# Capacity information of all workers
 $ ./bin/alluxio fsadmin report capacity
-#
-# Report metrics information
+# Capacity information of live workers
+$ ./bin/alluxio fsadmin report capacity -live
+# Capacity information of specified workers
+$ ./bin/alluxio fsadmin report capacity -workers AlluxioWorker1,127.0.0.1
+```
+
+`report metrics` will report the metrics information of Alluxio cluster.
+
+```bash
 $ ./bin/alluxio fsadmin report metrics
-#
-# Report under file system information
-$ ./bin/alluxio fsadmin report ufs 
 ```
 
-You can use `-h` to display helpful information about the command.
+`report ufs` will report the under file system of Alluxio cluster.
```

Comment:
add more detail on what about the UFS is reported. Perhaps show sample output from the example? Same for other commands here. If the output is long, we can abbreviate

Reply:
Add the sample results of `alluxio fsadmin report` and `report ufs`. 
For metrics, I don't think the sample result is useful and they are too long.
Thanks